images/dcos/bootstrap.sh

```diff
@@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
 etcd_server_port=${ETCD_SERVER_PORT:-4001}
 etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
 
-ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
-if [ "${ENABLE_ETCD_SERVER}" != true ]; then
-  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
-else
-  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
-  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
+ETCD_MESOS=${ETCD_MESOS:-disabled}
```

Comment:
`etcd_server_list` is always set, whether the proxy is listening there or the embedded server when ETCD_MESOS_FRAMEWORK_NAME is disabled.  If we want to support external clusters in the future we will want to conditionally set this, but that's out of scope for this change.